### DISCUSSION TITLE
How can I simply delete a file?

### DISCUSSION THREAD

**Discussion by ariel-frischer** [jesseduffield/lazygit#2394] [Q&A]
I can't seem to see any menu options for this? Editing the file and deleting all lines does not do what I want. I can unstage everything then stash it but that seems really bad. How?!

**Comment by mark2185** [comment]
What do you mean by `delete`, what would be the equivalent `git` command for what you want to achieve?

If you want to `discard` it (`git checkout -- <file>`), it's the `d` button.

**Reply by ariel-frischer** [reply]
Something like equivalent of `git rm myFile.txt`, what does that `discard` do exactly??

**Reply by mark2185** [reply]
`discard` does exactly `git checkout -- "/path/to/file"`, you can see it in the `Command Log` after executing.

As for `git rm`, it may very well be we don't have something that does _just_ that.
Try adding the file to `.gitignore` by pressing `i` and then choosing the `add to .gitignore` option, after which the file will be deleted, and discard the `.gitignore` file with `d`.
